viam_cartographer_test.go

```diff
@@ -100,10 +74,10 @@ func TestNew(t *testing.T) {
 		}()
 
 		attrCfg := &vcConfig.Config{
-			Sensors:       []string{"good_lidar"},
-			ConfigParams:  map[string]string{"mode": "2d"},
-			DataDirectory: dataDirectory,
-			DataRateMsec:  testDataRateMsec,
+			Camera:                map[string]string{"name": "good_lidar", "data_frequency_hz": testDataFreqHz},
```

Comment:
Can you please keep these & add back the tests which were testing the behavior without the feature flag?